src/routes/Plugin/Common/Selector.js

```diff
@@ -977,11 +987,159 @@ class AddModal extends Component {
+                                          addonBefore={<div style={{width: labelWidth}}>{item.label}</div>}
+                                          placeholder={placeholder}
+                                          key={fieldName}
+                                          type="number"
+                                        />)
+                                        }
+                                    </FormItem>
+                                  </Tooltip>
+                                </li>
+                              )
+                            } else if (item.dataType === 3 && item.dictOptions) {
+                              return (
+                                <li key={fieldName}>
+                                  <Tooltip title={placeholder}>
+                                    <FormItem>
+                                      {getFieldDecorator(fieldName, {
+                                        rules,
+                                        initialValue: defaultValue,
+                                      })(
+                                        <Select
+                                          placeholder={placeholder}
+                                          style={{ width: "100%"}}
+                                        >
+                                          {item.dictOptions.map(option => {
+                                            return (
+                                              <Option key={option.dictValue} value={option.dictValue==="true"?true:(option.dictValue==="false"?false:option.dictValue)}>
+                                                {option.dictName} ({item.label})
+                                              </Option>
+                                            );
+                                          })}
+                                        </Select>
+                                      )}
+                                    </FormItem>
+                                  </Tooltip>
+                                </li>
+                              )
+                            } else {
+                              return (
+                                <li key={fieldName}>
+                                  <Tooltip title={item.value}>
+                                    <FormItem>
+                                      {getFieldDecorator(fieldName, {
+                                        rules,
+                                        initialValue: defaultValue,
+                                      })(
+                                        <Input
+                                          addonBefore={<div style={{width: labelWidth}}>{item.label}</div>}
+                                          placeholder={placeholder}
+                                          key={fieldName}
+                                          onChange={e=> {
+                                            this.onDealChange(
```

Comment:
Why did you put two parameter  `item  and item.label `
```
onDealChange = (value,label,item) => {
    if(label === item.label){
```
Is  label === item.label alway right ?

Reply:
Maybe you can write more elegant

Reply:
Thanks,i have created a new pull request.